drivers/Driver.py

```diff
@@ -0,0 +1,5 @@
+# base class for the drivers inherit it to do own driver
+class Driver:
+    def __init__(self):
+        pass
+    # methods to discuss
```

Comment:
Według ustaleń z Prowadzącym:

- sterowniki powinny generować listę decyzji (VehicleOrientation?), które będą pobierane przez klasę Simulation i wykonywane w trakcie dotarcia na skrzyżowanie (aka koniec drogi). Można stworzyć konwencję, w której jeżeli nie będzie żadnej pozycji na liście w sterowniku, to zostanie wybrana losowa decyzja.
- pozostaje tylko pytanie kiedy te decyzje będą generowane (przez algorytmy SI). Raz w trakcie tworzenia sterownika (znając już początkową pozycję)?